docs/design/coreclr/botr/mscorlib.md

```diff
@@ -1,355 +1,356 @@
-Mscorlib and Calling Into the Runtime
+Mscorlib (a.k.a. `System.Private.CoreLib`) and calling into the Runtime
 ===
 
-Author: Brian Grunkemeyer ([@briangru](https://github.com/briangru)) - 2006
-
 # Introduction
 
-Mscorlib is the assembly for defining the core parts of the type system, and a good portion of the Base Class Library in .NET Framework. It has been renamed to System.Private.CoreLib in .NET Core, though many places in the code and documentation still refer to it as mscorlib. Base data types live in this assembly, and it  has a tight coupling with the CLR. Here you will learn exactly how & why mscorlib.dll is special, and the basics about calling into the CLR from managed code via QCall and FCall methods. It also discusses calling from within the CLR into managed code.
+`System.Private.CoreLib.dll` is the assembly for defining the core parts of the type system, and a good portion of the Base Class Library in .NET Framework. It was originally named `mscorlib` in .NET Core, though many places in the code and documentation still refer to it as `mscorlib`. This document will endeavour to stick to using `System.Private.CoreLib` or SPCL. Base data types live in this assembly, and it has a tight coupling with the CLR. Here you will learn exactly how and why SPCL is special and the basics about calling into the CLR from managed code via QCall and FCall methods. It also discusses calling from within the CLR into managed code.
 
 ## Dependencies
 
-Since mscorlib defines base data types like Object, Int32, and String, mscorlib cannot depend on other managed assemblies. However, there is a strong dependency between mscorlib and the CLR. Many of the types in mscorlib need to be accessed from native code, so the layout of many managed types is defined both in managed code and in native code inside the CLR. Additionally, some fields may be defined only in debug or checked builds, so typically mscorlib must be compiled separately for checked vs. retail builds.
+Since SPCL defines base data types like `Object`, `Int32`, and `String`, SPCL cannot depend on other managed assemblies. However, there is a strong dependency between SPCL and the CLR. Many of the types in SPCL need to be accessed from native code, so the layout of many managed types is defined both in managed code and in native code inside the CLR. Additionally, some fields may be defined only in Debug, Checked, or Release builds, so typically SPCL must be compiled separately for each type of build.
```

Comment:
I would use `CoreLib` as the short name. I think it is easier to understand and we are using that name much more frequently today.

E.g. I can see 2 instances of `SPCL` under CoreCLR, but several dozens of `CoreLib`.

Reply:
Out of curiosity, I have looked up possible SPCL meanings at https://acronyms.thefreedictionary.com/SPCL . From now on, we should call it "Synthetic Peptide Combinatorial Library" :-)

Reply:
Definitely changing to "CoreLib"...